client/ayon_blender/api/lib.py

```diff
@@ -251,20 +251,26 @@ def lsattrs(attrs: Dict) -> List:
         ):
             continue
         for node in getattr(bpy.data, coll):
-            for attr, value in attrs.items():
+            ayon_prop = node.get(pipeline.AYON_PROPERTY)
+            if not ayon_prop:
                 avalon_prop = node.get(pipeline.AVALON_PROPERTY)
                 if not avalon_prop:
                     continue
-                if (avalon_prop.get(attr)
-                        and (value is None or avalon_prop.get(attr) == value)):
+                else:
+                    node[pipeline.AYON_PROPERTY] = avalon_prop
+                    ayon_prop = avalon_prop
+                    del avalon_prop
```

Comment:
This is just guess, but I think it should be.
```suggestion
                    del node[pipeline.AVALON_PROPERTY]
```

NOTE: Could be the constants imported to avoid `pipeline.{CONSTANT}`?